{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-18699", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": []}}, "problemtype": {"problemtype_data": [{"description": []}]}, "references": {"reference_data": [{"url": "https://github.com/gopro/gpmf-parser/issues/43", "name": "https://github.com/gopro/gpmf-parser/issues/43", "refsource": "MISC", "tags": []}]}, "description": {"description_data": [{"lang": "en", "value": "An issue was discovered in GoPro gpmf-parser 1.2.1. There is an out-of-bounds write in OpenMP4Source in GPMF_mp4reader.c."}]}}
